runbotics/runbotics-scheduler/src/scheduler-database/credential-collection/credential-collection.service.ts

```diff
@@ -114,6 +114,25 @@ export class CredentialCollectionService {
         return collections;
     }
 
+    async findAllAccessibleForCredentials(user: IUser) {
+        const collections = await this.credentialCollectionRepository.find({
+            where: {
+                tenantId: user.tenantId,
+                credentialCollectionUser: {
+                    userId: user.id,
+                    privilegeType: PrivilegeType.WRITE,
+                },
+            },
+            relations: [...relations, 'credentials.createdBy', 'credentials.collection'],
```

Comment:
if we used the object argument instead of the array, we would have intellisense support